sigle/src/modules/publicStory/PublicStory.tsx

```diff
@@ -74,10 +81,36 @@ export const PublicStory = ({ story, settings }: PublicStoryProps) => {
           "& :where(a strong):not(:where([class~='not-prose'] *))": {
             color: safeSiteColor,
           },
+
+          px: '$4',
```

Comment:
PublicStoryContainer is already defining padding line 18, should this be moved there?